components/Base/Chips/Group/index.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="chips-group">
+    <button
+      v-for="item in values"
+      :key="item.value"
+      :class="{
+        'chips': true,
+        'chips--active': isActive(item.value)
+      }"
+      @click="onClick(item.value)"
+    >
+      <span :class="{ 'chips__label': isActive(item.value), 'chips__label--active': isActive(item.value)}">
+        {{ item.label }}
+        <img v-show="isActive(item.value)" width="14" height="14" src="@/assets/icons/ChecklistChips.svg" alt="ChecklistChips">
+      </span>
+    </button>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    values: {
+      type: Array,
+      default: () => [],
+      validator: (values) => {
+        return values.every(item => item.label && item.value)
+      }
+    },
+    mandatory: {
+      type: Boolean,
+      default: false
+    }
+  },
+  data () {
+    return {
+      active: null
+    }
+  },
+  mounted () {
+    this.initialValue()
+  },
+  methods: {
+    initialValue () {
+      const allValues = this.values.map(item => item.value)
+      const [semua] = allValues
+      this.active = semua
```

Comment:
Just curious, does this function set the `active` data to be the first item in the given array or to be the item that has `semua` value?

Reply:
Hello kak @bangunbagustapa, I want to set this function to set `active ` data to be the element that has `semua`  value and `semua` value is unique. Then, it is a coincidence the `semua` value is in the first element.

Reply:
Nice @doohanas, 

If the `active` data is setted to be the the element that has 'semua' value, this destructuring assignment below doesn't meet you needs

```js
const [semua] = allValues
```

The code above wants the first element in the array. The user that read or use this code will not understand it right away. If you want to make sure the 'active' data is setted to be 'semua', you can set it in the data

```js
data () {
  return {
    active: 'semua'
  }
}
```

or even better, you can set the `active` data to be the first element, so can just focus in the data you give to this component and set the first element in the array to be what you want as a initial value.

```js
data () {
  return {
    listChips: [
      {
        label: 'Should be my default',
        value: 'should be my default'
      },
      {
        label: 'Whatever',
        value: 'whatever'
      },
      {
        label: 'Whatever',
        value: 'whatever'
      }
    ],
  }
}
```

Reply:
I get it kak @bangunbagustapa. I want to set the `semua` value as `active` data for initial state. So, I put  `semua` value in first element.
 
`data () {
    return {
      listChips: [
        {
          label: 'Semua',
          value: 'semua'
        },
        {
          label: 'Masyarakat',
          value: 'masyarakat'
        },
        {
          label: 'Mitra',
          value: 'mitra'
        }
      ],`
     Then, because `semua` value in first element. I wrote this kak.
     `const [semua] = allValues`

Reply:
Thanks for the explanation, looks good to me @doohanas. 

Yes, my suggestion is just replace the variable name when you destructuring the array to be more general so it will be the default value regardless the name. As you mentioned above, `firstValue` is more general.

Reply:
Alright thank you kak @bangunbagustapa 🔥 🔥